gulpfile.js

```diff
@@ -615,6 +633,37 @@ function bundleTask (opts) {
   }
 }
 
+function configureBundleForSesify ({
+  browserifyOpts,
+  bundleName,
+}) {
+  // add in sesify args for better globalRef usage detection
+  Object.assign(browserifyOpts, sesify.args)
+
+  // ensure browserify uses full paths
+  browserifyOpts.fullPaths = true
+
+  // record dependencies used in bundle
+  mkdirp.sync('./sesify')
+  browserifyOpts.plugin.push(['deps-dump', {
+    filename: `./sesify/deps-${bundleName}.json`,
+  }])
+
+  const sesifyConfigPath = `./sesify/${bundleName}.json`
+
+  // add sesify plugin
+  browserifyOpts.plugin.push([sesify, {
+    writeAutoConfig: sesifyConfigPath,
+  }])
+
+  // remove html comments that SES is alergic to
+  const removeHtmlComment = makeStringTransform('remove-html-comment', { excludeExtension: ['.json'] }, (content, _, cb) => {
```

Comment:
So SES has some problem with HTML comments (even when they're embedded in JavaScript)? I'm be interested to hear more about whatever problem this is solving.

It should be harmless enough for this dependency visualization build I suppose

Reply:
I think it's related to this monster
```js
1 + <!-- 2
3
// => 4
```
I haven't encountered html comment openings in the wild, but I have encountered html comment closings `-->` as part of diagrams in comments. SES has a regex that throws if present. 

This isn't actually needed for the viz tho, but for sesify build runtimes. Part of this PR is sneaking in all the components for sesify so that the PR to actually enable it in prod will be quite small.

Reply:
I think the main reason it's blacklisted is that it's behavior is not standardized

Reply:
Welp - that does seem monstrous.

Maybe there's a way to make this transformation more targeted then. I'm concerned that if this transformation was used as-is for a proper sesify build, it could break HTML markup inserted via JavaScript.

Reply:
Yeah that's a good concern. Maybe it should warn when it encounters one.